### PR TITLE
Materials 04 e02

### PR DESCRIPTION
**Pull request by mariev**

### CODE REVIEW

**Comment by thebioengineer** [comment]
Content looks good - just need to add the requirements template to valtools? will approve after?

File: Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R
Comment (thebioengineer) [inline Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R:19]: To me this indicates that we should be able to provide "override" abilities to vt_validate_* or provide a separate function for rendering non-validation documents. vt_render_report() or something?

**Review by thebioengineer — CHANGES_REQUESTED** [review]

File: Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R
Comment (mariev) [reply Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R:19]: Perhaps this is getting into philosophical territory - I see the generation of requirements to be part of a validation process. From a purely technical perspective, I really do want to drive home that we are implementing a DRY system here - the requirements are a living source, but the pdfs are the snapshots that a validation process demands.

Comment (thebioengineer) [reply Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R]: I would agree with that - the reason we are considering partial renders are for inherently non DRY parts that an org might require - such as getting approval for requirements or test cases prior to actually executing them. An opportunity to break out of our framework that encourages DRY behavior may be a necessity

Comment (thebioengineer) [reply Materials/Materials-04-Introduction_to_Valtools/Materials-04-E02-Validation_Report/Materials-04-E02-answer-key.R]: But anyway, not something we need to resolve this moment and technically they can still just call rmarkdown::render or click "knit"

**Comment by mariev** [comment]
@thebioengineer do we have any blockers on this PR?

**Review by thebioengineer — APPROVED** [review]

**Comment by thebioengineer** [comment]
I think everything is 👍

**Pull request merged**
